### ISSUE TITLE
Inbound options: unknown domain strategy: Asis

### ISSUE THREAD

**Issue by AA1999** [MatsuriDayo/nekoray#763]
**Describe the problem**
When running a config on the latest version (3.14) I get the following error on startup of the application or any connection:

![image](https://github.com/MatsuriDayo/nekoray/assets/14325650/30815b53-6113-408e-a6b5-8e91b11dbacd)

Expected behavior:
Normal startup, normal connection (as it did in 3.12, the version that I had before this)

Actual behavior:
The attached error
**How to reproduce**
Switch to singbox core

Provide helpful screenshots, videos, text descriptions, subscription links, etc.

**Logs**

If you have logs, please upload them. Please see detailed steps for exporting logs within the documentation.
```

sing-box-extra: 1.3.4-1 NekoBox: nekoray-3.14-2023-07-25
2023/08/02 06:29:29 nekobox_core grpc server listening at 127.0.0.1:10484
>>>>>>>> Starting profile [VLESS] AA-czudy41b
<<<<<<<< Failed to start profile [VLESS] AA-czudy41b
>>>>>>>> Starting profile [VLESS] AA-czudy41b
<<<<<<<< Failed to start profile [VLESS] AA-czudy41b
>>>>>>>> Starting profile [VLESS] AA-czudy41b
<<<<<<<< Failed to start profile [VLESS] AA-czudy41b
>>>>>>>> Starting profile [VLESS] AA-czudy41b
<<<<<<<< Failed to start profile [VLESS] AA-czudy41b
```

**Comment by zhouxinghong** [comment]
With the world people

**Comment by AA1999** [comment]
I'm not sure what you mean

**Comment by es771993** [comment]
same

**Comment by mehdifirefox** [comment]
> Change `domain strategy` to anything non-empty, save, and change back. It should be fine, or uninstall, select sing-box directly for the first time.
> 
> But I want to know how this happened, can you guys reproduce this on a fresh install?

After the upgrade.

==
It wasn't a problem before.
=

I didn't understand your solution.
I didn't find this option in the program

**Comment by AA1999** [comment]
I only seem to get this after an update
3.12 shows no error regarding this but as soon as I upgrade to 3.14, this error appears

**Comment by ntflc** [comment]
> Change `domain strategy` to anything non-empty, save, and change back. It should be fine, or uninstall, select sing-box directly for the first time.
it works, thx

**Comment by ntflc** [comment]
> I only seem to get this after an update 3.12 shows no error regarding this but as soon as I upgrade to 3.14, this error appears
> 
> A fresh 3.14 with singbox (if you delete 3.12 entirely) does not show this error
> 
> > Change `domain strategy` to anything non-empty, save, and change back. It should be fine, or uninstall, select sing-box directly for the first time.
> 
> I'm unable to find this option in both the config and the application

in preferences - routing settings - common

**Comment by Reynarow** [comment]
> Change `domain strategy` to anything non-empty, save, and change back. It should be fine, or uninstall, select sing-box directly for the first time.
> 
> But I want to know how this happened, can you guys reproduce this on a fresh install?



> Change `domain strategy` to anything non-empty, save, and change back. It should be fine, or uninstall, select sing-box directly for the first time.
> 
> But I want to know how this happened, can you guys reproduce this on a fresh install?

 yeah thats work. go to preferences - routing settings - common and change once

**Comment by AA1999** [comment]
The weird thing is that for a fresh 3.14 domain strategy is empty and it's showing no issues with that

**Comment by Alirezaaraby** [comment]
> Change `domain strategy` to anything non-empty, save, and change back. It should be fine, or uninstall, select sing-box directly for the first time.
> 
> But I want to know how this happened, can you guys reproduce this on a fresh install?

after updating to version 3.26 this happened to me
